attelo/parser/intra.py

```diff
@@ -30,6 +30,56 @@ class IntraInterPair(namedtuple("IntraInterPair",
     pass
 
 
+def for_intra(dpack, target):
+    '''
```

Comment:
As a minor formatting issue, there are two conventions from PEP0257 that could be enforced here for a low cost:
* docstrings are enclosed by triple double quotes,
* multi-line docstrings begin with a summary line.